src/components/content/content-theme.scss

```diff
@@ -1,6 +1,4 @@
 md-content.md-THEME_NAME-theme {
   color: '{{foreground-1}}';
-  background-color: '{{background-default}}';
+  background-color: '{{background-color}}';
```

Comment:
Not sure, where this property exists? I think you're using our theming service wrong.